ws-nextjs-app/cypress/support/helpers/runTestsForPage.ts

```diff
@@ -0,0 +1,25 @@
+import settings from '../config/settings';
+
+export default ({ pageType, tests }) => {
+  const settingsMap = new Map(Object.entries(settings));
```

Comment:
Perhaps this test runner needs a list of URLs passed in, instead of looking up the generic settings here?